packets.go

```diff
@@ -49,8 +49,15 @@ func (mc *mysqlConn) readPacket() ([]byte, error) {
 			if data[3] > mc.sequence {
 				return nil, ErrPktSyncMul
 			}
-			return nil, ErrPktSync
+
+			//When MariaDB server is shutdown connection killed packet is sent
```

Comment:
Please put a space between `// and the comment.
Slightly changed comment: "The MariaDB server sends a packet with sequence number 0 when the server is shutdown."